app/views/users/show.html.erb

```diff
@@ -1,9 +1,18 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  </head>
-    <h1>Details of particular user</h1>
-  <body>
-  </body>
-</html>
\ No newline at end of file
+<!-- locals is a way to pass local variables to a partial -->
+<%= render partial: 'users/user', locals: { user: @user } %>
+<div>
+  <h3>Bio</h3>
+  <p><%= @user.bio %></p>
+</div>
+<hr><hr>
+
+<section id="posts-container">
+  <% @user.posts.limit(3).each do |post| %>
+    <%= render partial: 'posts/post', locals: { post: post } %>
```

Comment:
- [ ] It would be nice if you use the model method `recent_posts` instead of using the each loops 🤔 because there is not point of having these method if you don't utilize them and also they make your app efficient 👍